src/server/worldserver/worldserver.conf.dist

```diff
@@ -2690,6 +2690,22 @@ Battleground.TrackDeserters.Enable = 1
 
 Battleground.InvitationType = 0
 
+#
+#    Battleground.DisableQuestShareInBG
+#        Description: Disables the ability to share quests while in a Battleground.
+#        Default:     0  - (Disabled)
+#                     1  - (Enabled)
+
+Battleground.DisableQuestShareInBG = 0
+
+#
+#    Battleground.DisableReadyCheckInBG
+#        Description: Enables the ability to Ready Check while in a Battleground.
```

Comment:
can you make it a bit more explicit like "to send Ready Check survey" ?

Reply:
Oh fuck, I didn't change the description, it shouldn't be enabled, it should be "disabled" XD